flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/java/org/apache/flink/cdc/connectors/mysql/source/MySqlFullTypesITCase.java

```diff
@@ -141,6 +143,7 @@ public void testMySql8JsonDataTypesWithUseLegacyJsonFormat() throws Throwable {
 
     @Test
     void testMysql57TimeDataTypes() throws Throwable {
+        UniqueDatabase usedDd = fullTypesMySql57Database;
```

Comment:
Please double check if these changes are necessary, and revert these to keep this PR minimum.